components/konflux-workspaces/base/operator/config/manager/manager.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/instance: system
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: workspaces
+    app.kubernetes.io/part-of: workspaces
+    app.kubernetes.io/managed-by: kustomize
+  name: system
```

Comment:
the namespace name is changed to `konflux-workspaces` by kustomize, I just want to confirm with you that this is the namespace name you want to deploy all manager resources.

Reply:
That is what I think @sadlerap intended. So I will leave like that

Reply:
I will be more specific with namespace here I wonder why @sadlerap picked system here, do you know @filariow?

Reply:
system will be replaced anyway with kustomize. I just wanted to make sure you knew that effective namespace after kustomize build is konflux-workspaces

Reply:
I just want to be more explicit about the intend so it is clear

Reply:
moved back to system, lets move conversation to this PR https://github.com/redhat-appstudio/infra-deployments/pull/4040